sdk/core/core-http/lib/atomResourceSerializerBase.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Constants } from "./util/constants";
+import { serializeJsonToAtomXml } from "./util/xml";
+import { parseResultFromAtomResponse } from "./util/atomHandler";
+
+export class AtomResourceSerializerBase {
+  static serializeToAtomXmlRequest(
+    resourceName: string,
+    resource: any,
+    properties: string[],
+    xmlNamespace: string
+  ): string {
+    const content: any = {};
+    content[resourceName] = {
+      $: {
+        xmlns: xmlNamespace
+      }
+    };
+
+    if (resource) {
+      // Sort properties according to what is allowed by the service
+      properties.forEach((property: string) => {
+        if (resource[property] !== undefined) {
+          content[resourceName][property] = resource[property];
+        }
+      });
+    }
+
+    return serializeJsonToAtomXml(content);
+  }
+
+  static deserializeAtomResponse(nameProperty: string[], atomResponseInJson: any): any {
```

Comment:
It is clear from the function name that we are attempting to deserialize the json response from the service, but it is not clear what `nameProperty` represents here. Please add some jsdocs explaining the relevance of `nameProperty`

Reply:
Added docs